spansy/src/http/types.rs

```diff
@@ -347,6 +347,9 @@ impl Spanned for Response {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct Body {
```

Comment:
Can we change the `Body`'s doc comment to read
/// An HTTP request or response payload body.